Source/WebKit/WebProcess/Extensions/API/Cocoa/WebExtensionAPIPermissionsCocoa.mm

```diff
@@ -48,8 +48,6 @@
 {
     // Documentation: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/permissions/getAll
 
-    RELEASE_LOG(Extensions, "permissions.getAll()");
-
```

Comment:
i saw these got removed in one of your [PRs](https://github.com/WebKit/WebKit/pull/16854/commits/71338e8db0ce777a17016df4b4367a61f9261449#diff-c463bc70aecf160d7a810c45526f6f2f6b04bb4b80e0caab2c801056270e5f09) @xeenon ... not sure how they re-appeared

Reply:
Odd. Yeah the code generator does this now so removing them is correct.